packages/augur-ui/src/modules/market-charts/components/market-outcomes-chart/market-outcome-chart-highchart.tsx

```diff
@@ -120,7 +134,7 @@ export default class MarketOutcomesChartHighchart extends Component {
             snap: true,
             label: {
               enabled: true,
-              format: "{value:.4f} <span class='eth-label'>ETH</span>"
+              format: "{value:.4f} <span class='eth-label'>DAI</span>"
```

Comment:
Are we renaming classes to reflect the change? e.g. 'eth-label' ->'dai-label'.

Reply:
not at this time. We'll do that as part of the styling re-vamping